Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesExtensionsTest.kt

```diff
@@ -19,7 +20,7 @@ import testhelpers.BaseTest
 
 class PersonCertificatesExtensionsTest : BaseTest() {
 
-    private val time = Instant.parse("2021-06-24T14:00:00.000Z")
+    private val time = Instant.now()
```

Comment:
why? I think when time moves on the tests might fail

Reply:
We were using this time variable when mocking the vaccines. In the rules we are using `Instant.now()` to check whether the 14 days have passed and the tests actually failed using the old date. Why would the tests fail if time changes?

Reply:
Are not the dates in the certificates are mocked also? if you use `Now` and the conditions that are met now , won't be valid after after 4 months

Reply:
and I just checked the rules logic, you can pass whatever time you like to the rule

Reply:
But the dates in the certifcates are mocked using this time variable. 4 months from now, they would use the date 4 months from now 😃